source/draft_api.rst

```diff
@@ -600,7 +650,17 @@ Response body:
        "created_at":"2014-04-17",
        "updated_at":null,
        "deleted_at":null,
-       "revision":1
+       "revision":1,
+       "users": {
+         "members": [
+           "patcht"
+         ],
+         "spectators": [
+         ],
+         "managers": [
+           "tschuy"
+         ]
+       }
     }
 
 Note that this endpoint, when called, will automatically set the currently
```

Comment:
Only if he adds himself to spectators and members. Values don't propagate downwards, I believe we agreed. There may be overlap -- a person may be both a member and a spectator, or all three -- but this is not necessary.